client/src/components/Invitation.js

```diff
@@ -75,18 +75,81 @@ export default function FormDialog() {
 
   /*push emails to emailList*/
   const addEmail = () => {
-    // if (emailList.findIndex((email) => email == email) < 0)
+    if (!email) return;
     if (!emailList.includes(email)) {
       setEmailList([...emailList, email]);
-      setEmail("");
-    } else return;
+      setEmail('');
+    } else {
+      return;
+    }
   };
 
-  const submitEmail = async () => {
+  //handles email invitations
+  // maybe use a Material UI loading component?
+  const submitInvite = async () => {
     handleClose();
 
-    //need to grab current user id and email
-    sendInvite("5fad63358f96786e507a0b74", emailList);
+    let goodEmails = [];
+    let badEmails = [];
+
+    //separate each email into goodEmail or badEmail
+    for (const email of emailList) {
+      let res = await createInvite(email);
+      if (res) goodEmails = [...goodEmails, email];
```

Comment:
So my thinking was that the individual creation and then sending of emails would all happen on the back end, but I think this works fine.
But - we wouldn't want to call it good for any res, just if we get a status of 200 back.